src/server/services/openAI.ts

```diff
@@ -0,0 +1,26 @@
+import { Configuration, OpenAIApi } from 'openai'
+
+interface OpenAIData {
+  apiKey: string
+  model: 'gpt-3.5-turbo' | 'gpt-4'
+  role: 'user' | 'assistant' | 'system'
+  content: string
+}
+
+class OpenAIService {
+  async create({ apiKey, model, role, content }: OpenAIData) {
+    const openAI = new OpenAIApi(
+      new Configuration({
+        apiKey,
+      })
+    )
```

Comment:
make this run only once at class initialization

Reply:
> make this run only once at class initialization

I have updated it here https://github.com/kixlab/kuiz/pull/81/commits/b9185fd1247f2d9416bd6d83fe9b0497c58d26ed :)